buildenv.mk

```diff
@@ -42,7 +42,9 @@ EHSM_KMS_GIT_SHA=$(shell git rev-parse --short=7 --verify HEAD)
 ######## ehsm run folder  ########
 # the folder is used to store the files generated by the runtime
 # Please always keep the last /slash/
-RUNTIME_FOLDER := /var/local/ehsm/
+LOG_FOLDER := /var/log/ehsm/
+RUNTIME_FOLDER := /var/run/ehsm/
+LOCAL_DATA_FOLDER := /var/local/ehsm/
```

Comment:
suggest to add a prefix for each macro, like:
EHSM_LOG_FOLDER
EHSM_RUNTIME_FOLDER

Reply:
Thanks for suggesting. The changes are applied on the latest commit, please take a look.